GANDLF/utils/gandlf_logging.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import yaml
+from pathlib import Path
+from importlib import resources
+import tempfile
+from GANDLF.utils import get_unique_timestamp
+
+
+def _create_tmp_log_file():
+    tmp_dir = Path(tempfile.gettempdir())
```

Comment:
Just to remind about weird tmp path in Mac OS: https://stackoverflow.com/questions/847850/cross-platform-way-of-getting-temp-directory-in-python/43418319#43418319
Up to you if we want to use this system one, or use a workaround to use `/tmp` in macos also. (I'm slightly in favor of using `/tmp`, but that's quite subjective and I never actually was a real mac os user)

Reply:
> This is because on MacOS, i.e. Darwin, tempfile.gettempdir() and os.getenv('TMPDIR') return a value such as '/var/folders/nj/269977hs0_96bttwj2gs_jhhp48z54/T'; it is one that I do not always want.

Sorry, but I do not agree with this change. If someone is using macOS, they need to deal with their systems' idiosyncrasies, unless someone reports a proper issue.